packages/ui-components-react/src/icons/list.tsx

```diff
@@ -0,0 +1,149 @@
+// Copyright 2024 IOTA Stiftung.
+// SPDX-License-Identifier: Apache-2.0.
+import React from "react";
+import type { IconsProps } from "./iconsProps";
+
+const ListLight: React.FC<React.SVGProps<SVGSVGElement>> = props => (
+	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256" {...props}>
+		<rect width="256" height="256" fill="none" />
+		<line
+			x1="40"
+			y1="64"
+			x2="216"
+			y2="64"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="16"
+		/>
+		<line
+			x1="40"
+			y1="128"
+			x2="216"
+			y2="128"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="16"
+		/>
+		<line
+			x1="40"
+			y1="192"
+			x2="144"
+			y2="192"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="16"
+		/>
+	</svg>
+);
+
+const ListRegular: React.FC<React.SVGProps<SVGSVGElement>> = props => (
+	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256" {...props}>
+		<rect width="256" height="256" fill="none" />
+		<line
+			x1="40"
+			y1="64"
+			x2="216"
+			y2="64"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="24"
+		/>
+		<line
+			x1="40"
+			y1="128"
+			x2="216"
+			y2="128"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="24"
+		/>
+		<line
+			x1="40"
+			y1="192"
+			x2="144"
+			y2="192"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="24"
+		/>
+	</svg>
+);
+
+const ListBold: React.FC<React.SVGProps<SVGSVGElement>> = props => (
+	<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256" {...props}>
+		<rect width="256" height="256" fill="none" />
+		<line
+			x1="40"
+			y1="64"
+			x2="216"
+			y2="64"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="24"
+		/>
+		<line
+			x1="40"
+			y1="128"
+			x2="216"
+			y2="128"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="24"
+		/>
+		<line
+			x1="40"
+			y1="192"
+			x2="144"
+			y2="192"
+			fill="none"
+			stroke="currentColor"
+			strokeLinecap="round"
+			strokeLinejoin="round"
+			strokeWidth="24"
+		/>
+	</svg>
+);
+
+export const List: React.FC<IconsProps> = ({
```

Comment:
if we don't have a custom `IconsProps` type or interface, this works.
```suggestion
export const List = forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(({
```